models/clm/tutorial/README.rst

```diff
@@ -0,0 +1,1567 @@
++=========================+==============================================================+
+| CLM5_setup_assimilation | Main setup script that creates a CLM5-DART assimilation      |
+|                         | case. This script sets up ``case``, ``run`` and              |
+|                         | ``archive`` directories and stages the necessary files       |
+|                         | to the ``run`` directory to create a ``hybrid`` run.         |
+|                         | It uses traditional commands ``create_newcase``,             |
+|                         | ``case.setup``, ``preview_namelists`` and ``case.build``     |
+|                         | included as part of the CESM package.                        |
++-------------------------+--------------------------------------------------------------+
+| DART_params.csh         | The companion script to ``CLM5_setup_assimilation``          |
+|                         | that defines important case settings for CLM and DART.       |
+|                         | The majority of case setting edits occur within this script. |
++-------------------------+--------------------------------------------------------------+
+| CESM_DART_config        | Once the case is created, this script turns 'on'             |
+|                         | assimilation by providing links between the CLM and DART     |
+|                         | code. Converts a 'free' run into an assimilation run.        |
++-------------------------+--------------------------------------------------------------+
+| assimilate.csh          | This script is executed during the assimilation case         |
+|                         | run-time. It orchestrates communication between CLM and DART.|
+|                         | After the CLM forecast step is complete, it calls upon the   |
+|                         | DART executables to relay the CLM files (restart, history)   |
+|                         | to DART and executes the ``filter`` step to update the CLM   |
+|                         | state variable. These updated files are then reinserted      |
+|                         | into the restart file for the next CLM forecast step.        |
++-------------------------+--------------------------------------------------------------+
+| input.nml               | Contains DART specific namelist settings such as             |
+|                         | DA type, inflation, localization, outlier threshold etc.     |
++-------------------------+--------------------------------------------------------------+
+
+
+
+Step 4: Compiling DART
+----------------------
+
+Similar to CLM, it is necessary to compile the DART code before an assimilation
+can be performed.  The DART code includes a variety of build template scripts that provide
+the appropriate compiler and library settings depending upon your system environment.
+This is an example of how to set up the system environment for Cheyenne:
+
+::
+
```

Comment:
I think this is a good place to talk about modules. For instance, I had openmpi loaded. I compiled everything successfully but the run failed because `mpiexec_mpt` was expected instead of `mpirun`